### ISSUE TITLE
Use values from .env file in seed.json config

### ISSUE THREAD

**Issue by eablokker** [toymachiner62/node-mongo-seeds#47]
I want to use values from my .env file, and from process.env, in my seed config, but it is a JSON file so I can't use JavaScript variables in there.

Also seems like a bad idea to have this seed.json file committed to my git repo with my mongodb uris saved in it.

**Comment by rvalenciano** [comment]
JSON doesn't support env variables, the other solution is to allow also parse a node js module (with module.exports) where you can have variables and code. @toymachiner62 I have a similar requirement, maybe I can help you with it?

**Comment by toymachiner62** [comment]
I'd love your help on this.

**Comment by timelf123** [comment]
This is definitely needed for security. What do you want help with @toymachiner62? A simple fix could be changing `seed.json` to `seed.js`:

some pseudocode

```
import * as dotenv from 'dotenv';
dotenv.load({ path: '.env' });
const config = {
	undefined: process.env.MONGO_UNDEFINED || localhost:27017/MONGO_UNDEFINED,
	dev: process.env.MONGO_DEV || localhost:27017/dev,
	prod: process.env.MONGO_PROD || localhost:27017/production,
};

export dev;
```

**Comment by toymachiner62** [comment]
@eablokker Whether you commit your seed.json file is not a requirement of this module, but completely up to you. This module helps you generate that file, but if you dont' want it in source control by all means add it to your `.gitignore` or equivalent.

**Comment by eablokker** [comment]
@toymachiner62 I think I misstated the issue. I don't want to commit the config file with my database credentials. I want the config file to be able to get my database credentials from my .env environment variables.

**Comment by toymachiner62** [comment]
Got it. Working on something now.

**Comment by toymachiner62** [comment]
This was implemented in version 3.0.1 which is now also published to npm. Note this is a breaking change so see the [changelog](https://github.com/toymachiner62/node-mongo-seeds/blob/master/CHANGELOG.md)

**Comment by timelf123** [comment]
Thank you :champagne:
